src/parquet/column/column-reader-test.cc

```diff
@@ -155,5 +155,70 @@ TEST_F(TestPrimitiveReader, TestInt32FlatRepeated) {
   ExecuteDict(num_pages, levels_per_page, &descr);
 }
 
+TEST_F(TestPrimitiveReader, TestPageReader) {
```

Comment:
Can you change this test name to something descriptive, like "DictionaryEncodedPages"